subsystems/TypeHandlerLibrary/src/test/java/org/terasology/persistence/typeHandling/coreTypes/GenericMapTypeHandlerTest.java

```diff
@@ -32,6 +31,39 @@ GenericMapTypeHandler.VALUE, new PersistedLong(TEST_VALUE)
             ))
     ));
 
+    private final PersistedData testDataMalformatted = new PersistedValueArray(List.of(
```

Comment:
Maybe we can add the JSON equivalent we want to express here in a comment? Probably helps in both understanding the test case and figuring out if the data strucutre represents the thing we want to test.

Reply:
done :heavy_check_mark:

Reply:
:worried: It's also the textbook example of "comment that's out of date as soon as you change the code."

If you think it helps despite that, go ahead and keep it, but I'm a little concerned.

Also note Javadoc does not have triple-quote code blocks. Either change these to JavaDoc syntax, or do not start the comment with the double-asterisk `/**` so IntelliJ and javdoc don't treat it as javadoc.

Reply:
I've replaced the triple-quotes.